### PR TITLE
bump version for replace image

### PR DESCRIPTION
**Pull request by Daniel-Fan**
Not sure whether we need to update the version of replace image in 3.8 since there is a newer 3.7.2.

### CODE REVIEW

**Comment by Daniel-Fan** [comment]
We will hold it until the internal daily build for 3.8.0 moving to `v3` channel

File: bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -327,5 +327,5 @@ spec:
   maturity: alpha
   provider:
     name: IBM
-  replaces: ibm-common-service-operator.v3.7.1
+  replaces: ibm-common-service-operator.v3.7.2
```
Comment (bitscuit) [inline bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml:330]: Should be changed to 3.7.3 once it gets released

Comment (Daniel-Fan) [reply bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml]: Sure, update to `3.7.3`, and `3.8.0` has been move to `v3` channel, but let hold it until `3.7.3` is released.

**Comment by Daniel-Fan** [comment]
This PR is ready to merge

**Review by ZhuoxiLi — APPROVED** [review]
/lgtm

**Pull request merged**